src/app/api/permissions.py

```diff
@@ -32,3 +32,15 @@ def has_permission(self, request: Request, view: APIView) -> bool:
             return True
 
         return super().has_permission(request, view)
+
+
+class IsSuperUser(permissions.BasePermission):
+    message = _("Only superusers can perform this action.")
+
+    def has_permission(self, request: Request, view: APIView) -> bool:
+        return bool(request.user and request.user.is_superuser)
```

Comment:
1. bool is useless there
2. It's better to check that user is authenticated

```suggestion
        return request.user.is_authenticated and request.user.is_superuser
```